compiler/src/main/scala/org/scalajs/core/compiler/PrepJSInterop.scala

```diff
@@ -693,7 +694,7 @@ abstract class PrepJSInterop extends plugins.PluginComponent
         }
 
         val ownerLoadSpec = jsInterop.jsNativeLoadSpecOf(sym.owner)
-        val jsName = jsInterop.jsNameOf(sym)
+        val JSName.Literal(jsName) = jsInterop.jsNameOf(sym)
```

Comment:
It seems this is going to be much harder than I thought, because of the comment here: https://github.com/scala-js/scala-js/blob/51b537ae4c64783db016ca3e90e675a895d50992/compiler/src/main/scala/org/scalajs/core/compiler/PrepJSInterop.scala#L566-L569

I'm not sure yet how to make scalac do what we want around here ...